test/test_attn.py

```diff
@@ -0,0 +1,58 @@
+import torch
+
+from torch.testing._internal.common_utils import TestCase, run_tests
+
+def attn(q: torch.Tensor, k: torch.Tensor, v: torch.Tensor) -> torch.Tensor:
+    attn = q.matmul(k.t()).tanh()
+    return attn.matmul(v), attn
+
+class TestAttn(TestCase):
```

Comment:
Note that for this file to run in CI, you need to add it to the list in `test/run_test.py`

Reply:
done.